sdk/python/docsgen/types_doc.rst

```diff
@@ -57,8 +57,21 @@ Example use of YList type
 
 .. code-block:: python
 
-    from  from ydk.models.ietf.ietf_netconf_acm import Nacm
+    from ydk.models.ietf.ietf_netconf_acm import Nacm
 
     rule_list   = Nacm.RuleList()
     rule        = rule_list.Rule()
     rule.parent = rule_list
+
+Example use of YLeafList type
+=============================
+
+- The node being configured (ipv4_dscp) under the ydk.models.asr9k.Cisco_IOS_XR_asr9k_policymgr_cfg module:
+**ipv4_dscp:** An leaflist of Match IPv4 DSCP. **type:** YLeafList
+
+.. code-block:: python
+
+    from ydk.models.asr9k.Cisco_IOS_XR_asr9k_policymgr_cfg import PolicyManager
+
+    match = PolicyManager.ClassMaps.ClassMap.Match()
+    match.ipv4_dscp.extend(['15', '16'])
```

Comment:
Looks good. Can you add more examples of the features of YLeafList, including

1. splicing
2. only adding unique elements to leaf-list is allowed

?

Thanks